pkgs/tools/filesystems/rar2fs/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, fuse
+}:
+
+with stdenv.lib;
+
+let
+
+  version = "1.28.0";
+
+  unrar-src = fetchurl {
+    url = "https://www.rarlab.com/rar/unrarsrc-5.8.5.tar.gz";
+    sha256 = "0abvz6vv8kr416fphysfbwgxc6hyf1bpnd0aczfv7j3vc8x949d7";
+  };
+
+in
+
+stdenv.mkDerivation {
```

Comment:
You can use `stdenv.mkDerivation rec {` here (`rec` being the important change) and move `version` and `unrar-src` into the set below to avoid the `let ... in` (since you inherit both attributes anyway).

Reply:
Thanks! Will do.